src/sentry/incidents/logic.py

```diff
@@ -91,6 +91,7 @@ def create_incident(
         incident = Incident.objects.create(
             organization=organization,
             detection_uuid=detection_uuid,
+            # TODO: Should this function receieve the trigger, and we use the trigger type to set incident status?
```

Comment:
I think it might make sense to just pass the status while creating? I don't know that incident creation needs to know the specifics of why a status is set.

Reply:
Sounds reasonable to me. We can have status determining logic outside of this function. I can add a method to logic.py for determining the status of an incident at any given time, and we can use it when determining what the status on alert & resolution.